api/src/main/java/ru/urururu/sanity/api/cfg/ConstCache.java

```diff
@@ -1,6 +1,7 @@
 package ru.urururu.sanity.api.cfg;
 
 import org.springframework.stereotype.Component;
+import ru.urururu.sanity.cpp.Demangler;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'ru.urururu.sanity.cpp.Demangler'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)